reactor-aeron-core/src/main/java/reactor/aeron/DefaultAeronInbound.java

```diff
@@ -124,10 +134,8 @@ public void request(long n) {
 
     @Override
     public void cancel() {
-      REQUESTED.set(DefaultAeronInbound.this, 0);
```

Comment:
Why removed line 127?

Reply:
it's redundant now - we always check the subscriber first